src/store/wizardSlice.ts

```diff
@@ -660,6 +674,23 @@ export const wizardSlice = createSlice({
         1
       );
     },
+    addLanguage: (state, action: PayloadAction<string>) => {
+      if (!state.locale.languages?.some((lang) => lang === action.payload)) {
+        state.locale.languages?.push(action.payload);
```

Comment:
Ultra nitpick, do we need the optional chaining here (`?`)? I think we should only enter the `if` if state.locale.languages exists.

Reply:
Great point! Added the condition both in `addLanguage` and `removeLanguage` to get rid of the optional chaining.